apps/sms/index.html

```diff
@@ -148,6 +155,33 @@ <h1 data-l10n-id="messages"></h1>
           <a id="threads-settings-button" data-icon="settings" data-l10n-id="settingsButton"></a>
         </gaia-header>
         <article id="threads-container" class="view-body edit-container" data-type="list">
+          <gaia-fast-list id="threads-list" top="0" bottom="0" caching>
+            <!--<template header>
+              <gaia-sub-header data-time="${daySection}">${section}</gaia-sub-header>
+            </template>-->
+            <template item>
```

Comment:
Normally l20n.js handles data-l10n-id in a Mutation Observer, which should happen before any reflow; however it's right that this could bring frame skips.

I think we should start using them, and see how this looks...